rpm/kubelet.spec

```diff
@@ -161,7 +161,7 @@ mv cni-plugins/bin/ %{buildroot}/opt/cni/
 
 %files
 %{_bindir}/kubelet
-%{_sysconfdir}/systemd/system/kubelet.service
+%attr(644,-,-) %{_sysconfdir}/systemd/system/kubelet.service
```

Comment:
should we change `755` on the `install` line on L144 instead?

Reply:
That seems better to me too.